cmd/admin/admin.go

```diff
@@ -36,7 +36,7 @@ type admin struct {
 
 // newAdmin constructs a new admin object on the heap and returns a pointer to
 // it.
-func newAdmin(configFile string, dryRun bool) (*admin, error) {
+func newAdmin(configFile string, dryRun bool, debugAddr string) (*admin, error) {
```

Comment:
This line of diff can be undone, since admin doesn't need a debugaddr at all anymore.